addons/product_expiry/views/stock_quant_views.xml

```diff
@@ -48,7 +48,7 @@
             <xpath expr="//filter[@name='reserved']" position="after">
                 <separator/>
                 <filter string="Expiration Alerts" name="expiration_alerts"
-                    domain="[('removal_date', '&lt;=', context_today().strftime('%Y-%m-%d'))]"/>
+                    domain="[('alert_date', '&lt;=', context_today().strftime('%Y-%m-%d'))]"/>
```

Comment:
```suggestion
                    domain="[('lot_id.alert_date', '&lt;=', context_today().strftime('%Y-%m-%d'))]"/>
```

Reply:
@KangOl Done, thanks!
Could you, `r+` it, please ?